src/wepy/resampling/novelty/weight_novelty.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+from wepy.rsampling.novelty.novelty import Novelty
+
+class Weight_Factor_Novelty(Novelty):
+
+    def __init__(self, n_walkers, pmin=1e-12):
+        
+        # The minimum weight for a walker
+        self.pmin = pmin
+        
+        # log(probability_min)
+        self.lpmin = np.log(pmin / 100)
+
+        # The total number of walkers
+        assert n_walkers is not None, "Must give the total number of walkers"
+        self.n_walkers = n_walkers
+
+        self.clone_merge_iter = 0
+
+        # setting the weights parameter
+
+    def calculate_novelty(self, walkerwt, amp, images):
```

Comment:
docstring

Reply:
Added doc strings to the calculate_novelty method.